src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -9,39 +9,52 @@
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
 import java.io.IOException;
-import java.util.List;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.HashSet;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
+    Set<String> usersWithAccess = new HashSet<String>();
```

Comment:
Nice change, adding this to the backend!

Reply:
Thank you!